Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Set the base image to Ubuntu
+
+FROM selenium/standalone-chrome-debug
+
+# Necessary for HEADLESS
```

Comment:
Is this still a valid comment?  If so, what does it mean?

Reply:
No I guess it's not :/